src/Test.UnitTests.Sarif.Converters/TestData/MSBuildConverter/ExpectedOutputs/ValidResults.sarif

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "results": [
+        {
+          "ruleId": "MSB1003",
+          "level": "error",
+          "message": {
+            "text": "Specify a project or solution file. The current working directory does not contain a project or solution file."
+          }
+        },
+        {
+          "ruleId": "PV1002",
```

Comment:
>PV1002 [](http://example.com/codeflow?start=21&length=6)

should this also be `"level": "error",` ?
`ERROR: PV1002`

Reply:
bug fixed